<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_quickSpeedLimit</class>
 <widget class="QWidget" name="Form_quickSpeedLimit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>71</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>191</width>
    <height>71</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>191</width>
    <height>71</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>172</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_ulSpeedLimit">
      <property name="text">
       <string>上传限速</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_ulSpeedLimit">
      <property name="suffix">
       <string> KiB/s</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>2048</number>
      </property>
      <property name="singleStep">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_dlSpeedLimit">
      <property name="text">
       <string>下载限速</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_dlSpeedLimit">
      <property name="suffix">
       <string> KiB/s</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>2048</number>
      </property>
      <property name="singleStep">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_ulSpeedLimit</tabstop>
  <tabstop>spinBox_dlSpeedLimit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
